sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -185,20 +181,19 @@ private[sbtplugin] object ScalaJSPluginInternal {
         val output = (artifactPath in key).value
         val linker = (scalaJSLinker in key).value
         val usesLinkerTag = (usesScalaJSLinkerTag in key).value
+        val scalaJsOptTag = Tags.Tag(s"scalajs-${stage.tagName}")
```

Comment:
We write acronyms in all-uppercase when they are not first "word" of a camelCase identifier. So this would be `scalaJSOptTag`, for example.